networkx/algorithms/traversal/tests/test_beamsearch.py

```diff
@@ -25,3 +25,8 @@ def test_wide(self):
         G = nx.cycle_graph(4)
         edges = nx.bfs_beam_edges(G, 0, identity, width=2)
         assert list(edges) == [(0, 3), (0, 1), (3, 2)]
+
+    def test_none(self):
```

Comment:
could you be rename this to something like `test_width_none`

Reply:
Have done. Thanks for your advice.